### PR TITLE
PRS in Tanasee1 No. 7

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
PRS in Tanasee1 as discussed and revised in Pull requrest #742.

### CODE REVIEW

File: new/PRS14156Temhertu.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <change who="CH" when="2023-10-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ትምህርቱ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Təmhərtu</persName>
+                    <occupation type="ecclesiastic">qis gabaz</occupation>
```
Comment (eu-genia) [inline new/PRS14156Temhertu.xml:50]: qes

File: new/PRS14160TaklaMaryam.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <change who="CH" when="2023-10-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተክለ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Takla Māryām</persName>
+                    <occupation type="ecclesiastic">ʾaḫāyā ṭabotu</occupation>
```
Comment (eu-genia) [inline new/PRS14160TaklaMaryam.xml:50]: titles or jobs?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14102Estifanos.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-10-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <t:person xmlns:t="http://www.tei-c.org/ns/1.0" sex="1">
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS14102Estifanos.xml:47]: I found dating this addition on palaeographic grounds or any other reason would be too incertain. This is why I obstained from giving a date.

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS14102Estifanos.xml:47]: I removed t: and xmlns:t="http://www.tei-c.org/ns/1.0".

File: new/PRS14156Temhertu.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14156Temhertu.xml:50]: Yes.

File: new/PRS14160TaklaMaryam.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14160TaklaMaryam.xml:50]: I changed ʾaḫāyā ṭabotu and ʾato wanṣa to titles, unless this appears to me to be not very certain. Perhaps I should refrain to encode these terms at all.

Comment (eu-genia) [reply new/PRS14160TaklaMaryam.xml]: The syntax to me suggests that ʾaḫāyā ṭabotu is used in postposition to Tanśəʾa Krəstos

Comment (eu-genia) [reply new/PRS14160TaklaMaryam.xml]: I am not sure ወንጸ is a correct interpretation of what is written

Comment (eu-genia) [reply new/PRS14160TaklaMaryam.xml]: In any case ወ could well be a conjunction, and maybe there is a -ha missing or invisible at the end
maybe @DenisNosnitsin1970 can also have a look

**Review by eu-genia — APPROVED** [review]

**Pull request merged**